### PR TITLE
v2.5 Fixes/Updates

### LINKED ISSUE #1504: Missing boost jumps in the logic
As talked over by discord with Dyceron, I'm posting the videos of a couple of boost jumps we've got on video while playing in multiworld.

Crossing Ing Windchamber without space jump : https://www.youtube.com/watch?v=vJaPUYnCbck
Crossing Mining Station A from sand gate to Temple access without missile or space jump : https://www.youtube.com/watch?v=C5QS9CX5jQU&ab_channel=AlexDfhj

### LINKED ISSUE #1505: Way to reach the translator gate in sanc temple with only bombs, spider and scan visor
This trick is done in two part, the first consist on going on top of the sanc temple platform using the spider track to do a 2BJ. This first one is only useful if you do the second part so dont really matter which difficulty it is on.

https://user-images.githubusercontent.com/69166253/106966071-eb6a9580-6744-11eb-8d49-710977a325c4.mp4

Then after that an NSJ e-dash bended (havent test it well enough if it only work bended) to reach the translator door. This one is obivously a NSJ e-dash and if you fail you just fall off and have to do the first part again so because of that maybe expert difficulty.

https://user-images.githubusercontent.com/69166253/106966604-f40f9b80-6745-11eb-9666-1b1627136e8f.mp4

### LINKED ISSUE #1523: Reliquary Grounds -> Ing Reliquary only requires Light Suit
wIxXWdG7mpJ454b74jN1D0FF7D_WxT2FHDLU8SAjiAlAU4jAxMt6it5XboWWE0oiMUXG_hKTyllpvEIXRhSO0cDv

### LINKED ISSUE #1525: Path of Roots: Slope Jump to Pickup from Dark Door - Beginner -> Intermediate

### PR DESCRIPTION
**Pull request by dyceron**
- New Tricks added and adjustments made to logic
- Fixes #1504
- Fixes #1505
- Fixes #1523
- Fixes #1525

### CODE REVIEW

File: randovania/data/json_data/prime2.txt [hunk 1]
```diff
@@ -1947,14 +1948,19 @@ Asset id: 4237279396
   * Light Door to Reliquary Access/Door to Reliquary Grounds
   > Door to Ing Reliquary
       Any of the following:
-          Ingstorm Damage ≥ 1500
+          All of the following:
+              Ingstorm Damage ≥ 1500
+              Any of the following:
+                  Space Jump Boots
+                  Movement (Advanced) and Use Screw Attack (Space Jump)
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.txt:1955]: `Space Jump` or `SA & SJ`?

File: randovania/data/json_data/prime2.txt [hunk 2]
```diff
@@ -9854,6 +9861,9 @@ Asset id: 1815493844
           All of the following:
               Space Jump Boots
               Combat/Scan Dash (Intermediate) or Standable Terrain (Intermediate)
+          Morph Ball and Spider Ball and Morph Ball Bomb and Bomb Jump (Intermediate)
+  > Out of Bounds
+      Single Room Out of Bounds (Expert) and Use Screw Attack (No Space Jump)
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.txt:9866]: Is the NoSJ SA without Screw Attack at Z-Axis intended?

Comment (dyceron) [reply randovania/data/json_data/prime2.txt:9866]: That's below the zaxis, so it's unaffected

File: randovania/data/json_data/prime2.txt (hunk 1 above)
Comment (dyceron) [reply randovania/data/json_data/prime2.txt:1955]: Oh it's supposed to be nsj sa

**Pull request merged**